staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -596,7 +596,7 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintf(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
+			fmt.Fprintf(o.ErrOut, "Multiple resources found in %s namespace with the given name.\nUse different alias or the fully qualified resource name.", o.Namespace)
```

Comment:
This feels like a strange place for this to be but it seems like the right place?

Reply:
Do you mean 
- Import auth package and use it like `warningPrinter.Print(fmt.Sprintf(o.ErrOut, "Multiple resources found in %s namespace with the given name.\nUse different alias or the fully qualified resource name.", o.Namespace))`.

please confirm

Reply:
Don't import anything yet but print it out as "Warning: " so we can find it later.

Present the user with the possible collisions. I think there is history around this somewhere.